docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: '3.7'
+version: '3.1'
```

Comment:
Why is this one 3.1 instead of 3.2?

Reply:
@hillaryj, see the commit message for 90ae685.

Reply:
Perfect, thank you! I'll check the next time.